lib/webauthn/authenticator_data.rb

```diff
@@ -62,6 +62,14 @@ def attested_credential_data_included?
       flags.attested_credential_data_included == 1
     end
 
+    def backup_eligibility
```

Comment:
This could also be `backup_eligible?`. But `backup_eligibility` exactly matches the spec and flag name, so I think it would be cleaner to use that.

Alternatively, we could rename this to `backup_eligible?` and `backup_state` to `backup_state_synced?`, but I didn't feel so comfortable making an "editorial" change to the bit names. A lot of things in WebAuthn were originally meant to be one thing, but then developed a different meaning, and using the exact spec names reduces the chance of a stale assumption in the names used here.